lib/puppet/provider/package/windows/package.rb

```diff
@@ -67,7 +67,6 @@ def self.installer_class(resource)
         # REMIND: what about msp, etc
         MsiPackage
       when /\.exe"?\Z/i
-        fail(_("The source does not exist: '%{source}'") % { source: resource[:source] }) unless Puppet::FileSystem.exist?(resource[:source])
```

Comment:
It's possible `source` can refer to a local absolute path, such as an NFS mounted fileserver. So in that case, I think we still want to check if it exists. `Puppet::Util.absolute_path?` will work for that.

Reply:
Even though [docs](https://puppet.com/docs/puppet/7/resources_package_windows.html) have all examples with absolute paths, there is no restriction about using relative paths. We checked this and something like `puppet resource package '7-Zip 19.00 (x64)' ensure=installed source='relative_path/7z1900-x64.exe'` works as expected.
The only non-breaking check we found is ` || resource[:source].start_with?('http://', 'https://')`.